etc/config/mojo.amazon.properties

```diff
@@ -1,14 +1,11 @@
 compilers=&mojo
-defaultCompiler=mojo_nightly
-compilerType=mojo
+mojo.defaultCompiler=mojo_nightly
+mojo.compilerType=mojo
```

Comment:
This change shouldn't be necessary

Reply:
These changes came in as part of @mattgodbolt fixes when he published the live site. 8772251b4063d2163c289fcc773d49ffacf6214f

Reply:
Thanks!